docs/core/diagnostics/index.md

```diff
@@ -41,6 +45,14 @@ The [dotnet-gcdump](dotnet-gcdump.md) tool is a way to collect GC (Garbage Colle
 
 .NET Core includes what is called the `EventPipe` through which diagnostics data is exposed. The [dotnet-trace](dotnet-trace.md) tool allows you to consume interesting profiling data from your app that can help in scenarios where you need to root cause apps running slow.
 
+### dotnet-symbol
+
+[dotnet-symbol](dotnet-symbol.md) downloads files (symbols, DAC/DBI, host files, etc.) needed to open a core dump or minidump. Use this tool if you need symbols and modules to debug a dump file captured on a different machine.
```

Comment:
This tools is slightly more general purpose than this comment implies, but since we tend to be scenario centric perhaps this is OK.

Reply:
I can definitely broaden this a bit. What other scenarios should I be thinking about for the `dotnet symbol` summary? The only time I've used the tool was for getting debugging files needed for analyzing core dumps, so I'm not familiar with its other uses.

Reply:
dotnet-symbol can download symbols (PDBs, dbg) and binaries (DLLs, .so's) for all modules and assemblies in a dump.  It can also download pdbs/symbol files for a special module instead of a dump.  I don't think adding either is necessary for the summary; I think you captured the typical usage.

Reply:
Ah, I see. So, it can work on modules in addition to dumps. I'll leave it as is since it sounds like this is the typical usage, but that's good to know.

Reply:
So there were a few things things....

+ `debug` ...  this is also useful for profiling
+ `dump file`....  Live processes also can use this...
+ `on another machine` ... this is is also useful on the same machine to get the symbols.

I'm OK leaving as is...